src/main/java/com/cloudbees/jenkins/plugins/bitbucket/hooks/BitbucketSCMSourcePushHookReceiver.java

```diff
@@ -116,14 +120,16 @@ public HttpResponse doNotify(StaplerRequest2 req) throws IOException {
             String eventType = hookProcessor.getEventType(Collections.unmodifiableMap(reqHeaders), MultiMapUtils.unmodifiableMultiValuedMap(reqParameters));
 
             hookProcessor.process(eventType, body, context, endpoint);
+            listenersHandler.onProcess(eventType, body, endpoint);
```

Comment:
Failures such as https://github.com/jenkinsci/bitbucket-branch-source-plugin/blob/88e3abcd68367e3d6eb4e2e12f7ca6e77f76344c/src/main/java/com/cloudbees/jenkins/plugins/bitbucket/impl/webhook/cloud/CloudPushWebhookProcessor.java#L67 failing to parse the json(?) will still show success